docker/api/exec_api.py

```diff
@@ -50,7 +61,8 @@ def exec_create(self, container, cmd, stdout=True, stderr=True,
             'AttachStdin': stdin,
             'AttachStdout': stdout,
             'AttachStderr': stderr,
-            'Cmd': cmd
+            'Cmd': cmd,
+            'Env': environment or [],
```

Comment:
I was worried how `'Env': None` gets serialized to json, API spec said it should be array.

I trust you 🍸